src/workflow/ArcanistLandWorkflow.php

```diff
@@ -942,6 +942,82 @@ private function findRevision() {
         pht('Accepted'));
     }
 
+    // Check if all paths were reviewed by reviewers listed on METADATA files.
+    // If this check throws an exception - silently pass.
+    try {
+      $uber_metadata_unreviewed_paths = $this->getConduit()->callMethodSynchronous(
+        'uber_metadata.unreviewed_paths',
+        array(
+          'revisionid' => $rev_id,
+        ));
+      if (array_key_exists('pass', $uber_metadata_unreviewed_paths)
```

Comment:
At 966 line you use `isset($group['reviewers'][0]` to check key presence, looks like a bit inconsistent

Reply:
`isset` and `array_key_exists` behavior is different. `isset()` does not return TRUE for array keys that correspond to a NULL value, while `array_key_exists()` does. I wanted to be very precise of "pass" check while the "reviewers" array is just a helper to print information to the user.